Adding table parameter

krues8dr opened the pull request:
This is just scratching a small itch, we needed to use a specific migration table name, which `node-db-migrate` originally allowed but `sails-db-migrate` doesn't.  Can be set as a config file option or as a command line parameter (per the original).

leedm777 commented on lib/gruntTasks.js:
```diff
@@ -112,6 +120,7 @@ module.exports = function (grunt) {
     sails = new Sails();
     sails.lift(sailsConfig, function (err) {
       var url;
+      var args = buildDbMigrateArgs(grunt, sails, command);
```
Instead of passing the entire `sails` object in, please pass in `sails.config.migrations`. Building the database args shouldn't need anything more than the migrations config.

leedm777 commented on lib/gruntTasks.js:
```diff
@@ -10,7 +10,7 @@ var sailsDbMigrate = require('./sailsDbMigrate');
  * @returns Arguments array for the db-migrate command.
  */
-function buildDbMigrateArgs(grunt, command) {
+function buildDbMigrateArgs(grunt, sails, command) {
```
With the change recommended below, `sails` would change to `config`. Please add it to the JsDoc.

leedm777 commented on lib/gruntTasks.js:
```diff
@@ -43,12 +43,21 @@ function buildDbMigrateArgs(grunt, command) {
+    args.push('--table');
+    args.push(grunt.option('table'));
+  }
+  else if (sails.config.migrations.table) {
```
This is good thinking, allowing the table to be set in the config. I should probably do that for `coffee-file` and `migrations-dir`, too.

leedm777 commented on README.md:
````diff
@@ -34,6 +34,18 @@ module.exports.migrations = {
 };
 ```
 
+Optionally, you can specify in the config file the name of the database table to
````
Thanks for updating the README!

leedm777 commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sails-db-migrate",
-  "version": "0.8.0",
+  "version": "0.8.1",
```
You don't need to bump the version number. I'll do it when I publish the release.

leedm777 commented on lib/gruntTasks.js:
```diff
@@ -43,12 +43,21 @@ function buildDbMigrateArgs(grunt, command) {
+  if (grunt.option('table')) {
+    args.push('--table');
+    args.push(grunt.option('table'));
+  }
```
I prefer the else to be on the same line as the closing brace. `} else {`

leedm777 commented:
I've got a handful of nits I'd like to see addressed. Please address them, and rebase to a single commit.

It's a great addition. Thanks!

krues8dr commented:
Great, I cleaned those up, and squashed it down.  I also went ahead and added the two extra config options (`coffeeFile` and `migrationsDir`) as well.  Please let me know if this works!

leedm777 commented:
Looks great. Thanks!

leedm777 commented:
Published in v1.0.0

krues8dr commented:
Fantastic, thanks so much!

leedm777 commented:
@krues8dr Thank you!
